test/python/pulse/test_pulse_lib.py

```diff
@@ -27,6 +26,7 @@
     gaussian_square,
     drag as pl_drag,
 )
+from qiskit.pulse.library.symbolic_pulses import EnvelopeDescriptor, ConstraintsDescriptor
```

Comment:
I think testing with subclass covers everything. Test for descriptor is also added (it is tested by instantiation of pulse subclass).

Reply:
Do we have tests on Symbolic pulses and parameters?

Reply:
Since this PR replaces import pass of `Gaussian`, `GaussianSquare`, `Drag`, `Constant` from one of `ParametricPulse` version to `SymbolicPulse`. The existing tests are applied to `SymbolicPulse` now and everything is covered with them. This means test for `ParametricPulse` has been dropped. I'm not sure if we need to duplicate all the tests. Since this appears in multiple unittest files not only in pulse but also in transpile, qobj, etc...